compiler/q-implant/README.md

```diff
@@ -0,0 +1,25 @@
+# q-implant
+
+_q-implant_ writes quantization parameters and weights (given as .json and .npy files) to a circle model.
```

Comment:
```suggestion
_q-implant_ embeds quantization parameters and weights (given as .json and .npy files) into circle model.
```

Reply:
plz explain more about "writes weights (.npy)": where does this .npy file come from?

Reply:
Hello, I'm Junyeong Kang working with @FantBlog. 

Answer 1.

I'm sorry.
This README.md is already merged, but we made a mistake of not matching sync. #11562
We'll post a separate PR to revise README.md by supplementing the feedback section

Answer 2.

> plz explain more about "writes weights (.npy)": where does this .npy file come from?

We create ".npy" files through "[test_utils.py](https://github.com/Samsung/ONE/pull/11579/files/cca7de5fe6654722292d274b5f9c261f2f96769a#diff-c855e669b0b66191648c521df67c731f0b1dc738497894cb0ef5cc5cabc5c3d1R28)". 

".npy" is a numpy-type file used to represent data from quantization parameters and weights in param.json.

We write quantization parameters and weights in the form of ".npy" into the tensor through "q-implant".

Reply:
`test_utils.py` uses random value generation. we use random value for testing and for testing, `test_utils.py` looks OK.
but for real model quantization, how do we get this npy files?

Reply:
`q-implant` is a module that only writes quantization parameters and weights into the tensor of the existing circle. 
It is correct to use `test_utils.py` to verify this behavior.

Therefore, if users want to use `q-implant` for the real model, they must obtain qparam (param.json + .npy) through the extraction process.

That's why our team tries "q-extract-torch" module.

Reply:
> Therefore, if users want to use q-implant for the real model, they must obtain qparam (param.json + .npy) through the extraction process.

Then, `q-implant` should provide the structure / specification of this json and npy file.

Reply:
> > Therefore, if users want to use q-implant for the real model, they must obtain qparam (param.json + .npy) through the extraction process.
> 
> 
> 
> Then, `q-implant` should provide the structure / specification of this json and npy file.
> 
> 
> 
> 

As soon as possible, we create pull request about 'q-implant' README.md including previous feedbacks and the structure / specification of json and npy file.

Thank you so much!

Reply:
I don't think #11638 provides specification for q-implant json file.
Please be more specific.
Update this PR before posting the PR.

Reply:
> I don't think #11638 provides specification for q-implant json file. Please be more specific. Update this Draft PR before posting new PR.

OK. I updated the PR to Draft PR.
I'll update supplementary explanations for q-implant json file soon.